5_recognition.ipynb

```diff
@@ -274,14 +486,27 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "閾値を下限130、上限140ぐらいに設定すると安定して抽出できるようです。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "It seems that stable extraction can be performed by setting the threshold to the lower limit of 130 and the upper limit of 140."
```

Comment:
"It seems that stable extraction can be achieved by setting the threshold to the lower limit of 130 and the upper limit of 140."